lib/full-site-editing/edit-site-export.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * REST endpoint for exporting the contents of the Edit Site Page editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Output a ZIP file with an export of the current templates
+ * and template parts from the site editor, and close the connection.
+ */
+function gutenberg_edit_site_export() {
+	// Create ZIP file and directories.
+	$filename = tempnam( get_temp_dir(), 'edit-site-export' );
+	$zip      = new ZipArchive();
```

Comment:
`ZipArchive` isn't a required class, we should check if it exists before using it.

Reply:
This is outside of the scope of this PR. I didn't write this code. I'll see if it's a simple change.

Reply:
Ah, my mistake I thought it was new code.

I think doing a `if ( ! class_exists() ) return new WP_Error` should be sufficient.